README.md

```diff
@@ -57,7 +56,7 @@ We recommend installing Rust through [rustup](https://www.rustup.rs/). If you do
   $ curl https://sh.rustup.rs -sSf | sh
   ```
 
-  Parity Ethereum also requires `gcc`, `g++`, `pkg-config`, `file`, `make`, and `cmake` packages to be installed.
+  Open Ethereum also requires `clang`, `clang++`, `pkg-config`, `file`, `make`, and `cmake` packages to be installed.
```

Comment:
we've switched to `clang` sometime ago

Reply:
We could even state, that `OpenEthereum` requires `clang 9`. We had some reported problems with earlier versions when building `rocksdb`